force-app/main/default/objects/ServiceSchedule__c/fields/AttendanceRate__c.field-meta.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>AttendanceRate__c</fullName>
```

Comment:
This field is not listed in the AC's do we want to verify we should add it and if yes should we then also add it to the layout?

Reply:
Also one more thing I see is that we are not displaying Attendance Rate field on the page layout for Service Session, Program Engagement, Service and Contact objects where we are calculating Attendance.

Reply:
Good call, probably would follow the same pattern. Let's add the AC if Meg wants does want that field as well but I assume you are probably correct.